backend/src/main/java/com/bakdata/conquery/sql/execution/SqlExecutionService.java

```diff
@@ -76,20 +85,24 @@ private String getColumnName(ResultSet resultSet, int columnIndex) {
 		}
 	}
 
-	private Object[] getResultRow(ResultSet resultSet, int columnCount) {
-		// JDBC ResultSet indices start with 1 and we skip the first column because it contains the id
-		return IntStream.rangeClosed(2, columnCount)
-						.mapToObj(columnIndex -> this.getValueOfColumn(resultSet, columnIndex))
-						.toArray();
-	}
+	private SqlEntityResult getResultRow(ResultSet resultSet, List<ResultType> resultTypes, int columnCount) throws SQLException {
 
-	private String getValueOfColumn(ResultSet resultSet, int columnIndex) {
-		try {
-			return resultSet.getString(columnIndex);
-		}
-		catch (SQLException e) {
-			throw new ConqueryError.SqlError(e);
+		int rowNumber = resultSet.getRow();
+		String id = resultSet.getObject(PID_COLUMN_INDEX).toString();
+		Object[] resultRow = new Object[columnCount - 1];
+
+		CDateSet validityDate = this.resultSetProcessor.getCDateSet(resultSet, VALIDITY_DATE_COLUMN_INDEX);
+		resultRow[0] = validityDate;
+
+		int resultTypeIndex = 0;
+		for (int resultSetIndex = VALUES_OFFSET_INDEX; resultSetIndex <= columnCount; resultSetIndex++) {
+			int resultRowOffset = resultTypeIndex + 1;    // validity date is part of result row, but not part of the result type list
```

Comment:
In `com.bakdata.conquery.apiv1.query.ConceptQuery#getResultInfos` ist das Teil der resultInfos